index.html

```diff
@@ -358,68 +470,84 @@ <h2>
     </section>
     <section data-dfn-for="Navigator">
       <h2>
-        Navigator Interface extension
+        Extensions to the {{Navigator}} interface
```

Comment:
Just "code" (backticks) for headings is ok... the Navigator object will be hyperlinked in the IDL:
 
```suggestion
        Extensions to the `Navigator` interface
```